app/reducers/lnd.js

```diff
@@ -332,7 +332,11 @@ export const startActiveWallet = () => async (dispatch, getState) => {
   if (!state.lnd.lndStarted && !state.lnd.startingLnd) {
     const activeWalletSettings = walletSelectors.activeWalletSettings(state)
     if (activeWalletSettings) {
-      dispatch(startLnd(activeWalletSettings))
+      try {
```

Comment:
Not sure about this.

The reason why I'm wrapping the call to `await dispatch(startLnd(activeWalletSettings))` in a `try/catch` is because `startLnd` can return a rejected promise, and rejected promises should always be handled and will result in a message in the console about unhandled promise rejections if they are not.

In this particular case, I don't care about the result of `startLnd` and want to suppress warnings about unhandled promise rejections- which is what the `try/catch` is for.

But I don't really want to suppress other errors that could indicate a problem with the code.

Reply:
I see. In this case some comment would be good for future contributors to know what's going here

Reply:
Fair point!

Ok, so I've decided to change how we handle this slightly. Now, rather than swallowing the error in the `startActiveWallet` I have pushed this further down the chain so that we handle it where `startActiveWallet` actually called.

Ignoring the error from `startLnd` is specific to where we are calling `startActiveWallet` in this particular instance, and whilst we currently only use this method in this one place, that may change in the future and it should really be up to the caller of `startActiveWallet` how they handle the error case.

See https://github.com/LN-Zap/zap-desktop/pull/1261/files#diff-aad4104dcb0ed4f1a6fff1b59f88173eR67

Reply:
Looks good. The only things is that I would explicitly state that you are intentionally swallowing an error here `catch(() => ({}))`For a moment I thought that  you are returning an empty error and catching it  and handling it somewhere else as per comment

Reply:
haha, yes that's a basic ES6 noop. Have updated the comment for clarity:

```
// Catch the error and swallow it with a noop.
// Errors are handled below by listening for updates to the startLndError prop.
```